src/utils/autorestOptions.ts

```diff
@@ -36,6 +36,7 @@ export async function extractAutorestOptions(): Promise<AutorestOptions> {
   const batch = await getBatch(host);
   const multiClient = await getMultiClient(host);
   const generateSample = await getGenerateSample(host);
+  const productDocLink = await getProductDocLink(host);
```

Comment:
please add an example in the smoke test to show productDocLink are actually working.

Reply:
Will do that